modules/Emails/javascript/EmailUI.js

```diff
@@ -2804,6 +2805,8 @@ SE.folders = {
     	for(i=0;i<a_rs.length;i++)
     	{
     		var t_record = a_rs[i];
+          // if it's not working try something like this:
```

Comment:
Something about this comment smells bad. Has this been tested? It doesn't emit confidence.

Perhaps consider using a js doc in front of the code instead of this comment? 
```
  /**
   * @see SE.folders.setFolderSelection()
   */
```